xlators/mgmt/glusterd/src/glusterd-gfproxyd-svc.c

```diff
@@ -141,7 +140,6 @@ glusterd_gfproxydsvc_create_volfile(glusterd_volinfo_t *volinfo)
     xlator_t *this = NULL;
 
     this = THIS;
-    GF_VALIDATE_OR_GOTO("glusterd", this, out);
```

Comment:
nit: please fix the 'if' in line 152 ('this ? this->name' part in the gf_msg_debug() call)  (this is happening in other functions as well)

nit2: we don't need the 'out' label anymore.

Reply:
Ack.  WIll make the changes accordingly..